lib/guard/standardrb.rb

```diff
@@ -4,15 +4,21 @@
 
 module Guard
   class Standardrb < Plugin
-    def initialize(options = {})
+    attr_reader :fix
+
+    def initialize(opts = {})
       super
-      @options = options
+      @fix = opts[:fix]
     end
 
     def start
+      Guard::Compat::UI.info "Standardrb --fix = #{fix}"
```

Comment:
Perhaps have this line follow the other info line? It makes more sense to me that it will _inspect & then fix_ rather than determine the status of fix, and then inspect.

Reply:
yeah gotcha will change that